index.bs

```diff
@@ -753,6 +773,54 @@ authorizing an authenticator.
 </div>
 
 
+#### Construct the resultant credential - `constructResultantCredentialCallback` method #### {#constructResultantCredentialCallback}
```

Comment:
Should this really be part of this PR?

Reply:
this is probably a leftover from a previous merge. I will take it out.

Reply:
you don't need to take it out if you merge from current master (which I think you did in a commit more recent than this one here)